baseplate/metrics.py

```diff
@@ -417,27 +429,27 @@ def replace(self, new_value):
         .. note:: Due to the way the protocol works, it is not possible to
             replace gauge values with negative numbers.
 
-        :param float new_value: The new value to store in the gauge.
+        :param new_value: The new value to store in the gauge.
 
         """
         assert new_value >= 0, "gauges cannot be replaced with negative numbers"
         serialized = self.name + (":{:g}|g".format(new_value).encode())
         self.transport.send(serialized)
 
 
-def make_client(namespace, endpoint):
+def make_client(namespace: str, endpoint: EndpointConfiguration) -> Client:
```

Comment:
Why isn't `Client` in quotes here like some of the other functions?  Typo or quirk of python type hints?

Reply:
because client is defined by this point so we can use the actual type instead of the quotes for a forward declaration

Reply:
(on py3.8 or 3.7 w/ `from __future__ import annotations` annotations are done in a second pass and don't have this restriction)

Reply:
Ahh, thanks for the info :)